### PR TITLE
making cassandra framework resources configurable

### PR DESCRIPTION
**Pull request by kensipe**

### CODE REVIEW

**Comment by kensipe** [comment]
@elingg looking for review on this...  I am looking to test still... back in a few with the results.

**Comment by elingg** [comment]
LGTM assuming this is tested

**Comment by elingg** [comment]
Possibly this could be an append only change and we should create a new folder for this, however, I feel it is quite small. I could go either way.

**Comment by kensipe** [comment]
tested good!!

File: repo/packages/C/cassandra/1/config.json [hunk 1]
```diff
@@ -31,6 +31,23 @@
...
+              "description": "CPU shares to allocate to each Cassandra framework instance.",
+              "type": "number"
+            },
+            "mem": {
+              "default": 512.0,
+              "description": "Memory (MB) to allocate to each Cassandra framework instance.",
+              "minimum": 512.0,
+              "type": "number"
+            },
+            "instances": {
```
Comment (BenWhitehead) [inline repo/packages/C/cassandra/1/config.json]: This should not be configurable.  We only ever want one instance of the scheduler running at a time.  The number of nodes in the cluster is configured by another parameter.

File: repo/packages/C/cassandra/1/marathon.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
   "id": "/cassandra/{{cassandra.cluster-name}}",
   "cmd": "$(pwd)/jre*/bin/java $JAVA_OPTS -classpath cassandra-mesos-framework.jar io.mesosphere.mesos.frameworks.cassandra.framework.Main",
-  "instances": 1,
-  "cpus": 0.5,
-  "mem": 512,
+  "instances": {{cassandra.framework.instances}},
```
Comment (BenWhitehead) [inline repo/packages/C/cassandra/1/marathon.json]: See not above in config.json

**Comment by elingg** [comment]
Oh I missed this. yes we want only once instance of the scheduler at this point cc @kensipe. Making mem and cpus should be configurable though.

**Pull request merged**